### ISSUE TITLE
import stuck at 100% and not moving

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2834]
After uploading with no problem, the import of a BUAP workstation got stuck at 100% and never moved on.
![image](https://github.com/user-attachments/assets/e408f363-d430-44a7-9df4-1057ce9f42c3)
I got impatient and told it to synchronize anyway, and now THAT is stuck at initialization. So I suspect a server problem.

**Comment by lbestock** [comment]
Aha synchronization finally started. And indeed, nothing was back from the field and so nothing synchronized.

**Comment by urapadmin** [comment]
ja, the original import process got aborted. Not sure why, yet. I just try again,

**Comment by urapadmin** [comment]
okay, looks like FileMaker ist stuck. That piece of trash.,

**Comment by urapadmin** [comment]
on it...

**Comment by lbestock** [comment]
OK. Livia will need to download again later today to keep working independently, so update the ticket when it's ready to go.

**Comment by urapadmin** [comment]
FileMaker is indeed responding on Beset with a message that I have never seen before. I have to debug that. 
But I made the import go through. Please look at the synchronization log NOW if it is okay.

**Comment by urapadmin** [comment]
Does not look great to me.
![image](https://github.com/user-attachments/assets/25025f87-98d7-4b5f-916e-6a33e7ebd90b)

**Comment by urapadmin** [comment]
were you using FileMaker on an iPad or on a Mac? 
Can you tell me what version of FileMaker you were using? (you can see that when you open a Kiosk recording app in FileMaker and click on the big black Info Button in the ABOUT tab on the start page.

**Comment by lbestock** [comment]
What's up with the "temp" stuff? Otherwise the amount and nature of things on the log looks about right, and the auto collected material attachment is so great. @liviah848 was on her Mac with a newly installed FM, but I'll let her tell you which version.

**Comment by lbestock** [comment]
It looks proper when I download again. The things Livia input are there.

**Comment by lbestock** [comment]
Ah, no. The photos are broken.

**Comment by liviah848** [comment]
perhaps I input the photos wrong? Could this be the reason behind the issues? Also I am FileMaker Pro version 21.0.1.53.

**Comment by urapadmin** [comment]
Ah., That's at least a FileMaker version we have not used so far. I doubt that that is the issue, but It might be a hint.
Sorry, but there is no quick fix for this, as I have never seen it, so I need this evening to look into the issue and until then it might be better that you wait with further work. I'll give you the go as soon as I know what's going on.

**Comment by urapadmin** [comment]
> What's up with the "temp" stuff? Otherwise the amount and nature of things on the log looks about right, and the auto collected material attachment is so great. @liviah848 was on her Mac with a newly installed FM, but I'll let her tell you which version.

temp stuff? If you mean "2 new lines inserted from temp_ ..."? That has always been there and is fine. Just a compromise between what a developer needs to know and what might still be a useful information for an admin.

**Comment by urapadmin** [comment]
> perhaps I input the photos wrong? Could this be the reason behind the issues? 

No, don't worry. You can't put anything in wrong. If the user interface let you put in something the wrong way it would still be a bug of the user interface. But I don't think that is the problem. I was able to "see" the images in the database, so I will be able to recover them.

**Comment by urapadmin** [comment]
The current state of this is that I cannot access the images with FileMaker 20.3.3 on Windows. 
I can access those images with FileMaker 21 on Mac. What I don't know is if the issue is the Mac or the FileMaker version. 
For that I would need access to FileMaker 21 for Windows but Brown does not offer this version for Windows. 

But at least I have been able to get those two images out with FM21 and so I updated the broken images in Kiosk. That's healed now,
